src/app/views/app-sections/AppTitle.tsx

```diff
@@ -24,7 +24,7 @@ export function appTitleDisplayOnFullScreen(
             ariaLabel='Minimize sidebar'
             onClick={() => toggleSidebar()} />
       </TooltipHost>
-      <div className={classes.graphExplorerLabelContainer}>
+      <div className={classes.graphExplorerLabelContainer} role={'heading'} aria-level={1}>
```

Comment:
is this change for an accessibility blocking bug?

Reply:
this was a bug caught by the test..